packages/waku/src/router/create-pages.ts

```diff
@@ -147,6 +147,18 @@ export type CreateLayout = <Path extends string>(
       },
 ) => void;
 
+type Method = 'GET' | 'POST' | 'PUT' | 'DELETE';
+
+export type CreateApi = <Path extends string>(params: {
+  path: Path;
+  render: 'static' | 'dynamic';
```

Comment:
I'm not sure if we call `render` for api.

Reply:
changed to `mode` for now... let me know what you think

Reply:
I'm neutral. @sandren may have an idea.